terraform/warehouse/vcl/main.vcl

```diff
@@ -136,20 +136,24 @@ sub vcl_recv {
     # thing, reducing cache misses due to ordering differences.
     set req.url = boltsort.sort(req.url);
 
-    # Strip Accept-Language headers from request, the backends currently attempt
-    # to respect this, but we're not setup for it yet... in VCL or otherwise.
-    # This leads to us caching translated content in the default locale (en).
-    unset req.http.Accept-Language;
```

Comment:
Don't we still need to unset this somewhere?

Reply:
No, we pass Accept-Language along with the value found in the lookup, or `en` by default. This is what allows the users browser preference to work.

If a user sets a cookie, it will override Accept-Language.

Reply:
Isn't this still going to still potentially Vary in the case where `Accept-Language` is something we don't support and the response ends up being `en`?

Reply:
[`accept.language_lookup`](https://developer.fastly.com/reference/vcl/functions/content-negotiation/accept-language-lookup/) covers us.

If an `Accept-Language` value isn't negotiated, it will give `en` as the value.

Reply:
Discussed offline, makes sense now.